resources/views/admin/blocks/image_slider.blade.php

```diff
@@ -1,3 +1,8 @@
+@php
+    $currentUrl = explode('/', request()->url());
+    $type = $currentUrl[5] ?? null;
```

Comment:
We might consider making this a helper function.

Reply:
Good call^ 

It's also worth mentioning that once we upgrade to Twill 3 we should be able to use [Block Classes](https://twillcms.com/docs/block-editor/block-classes.html) in each of these blocks to make them context aware